### ISSUE TITLE
Streaming service not working even with SPOOF_ALL_DOMAINS

### ISSUE THREAD

**Issue by stickperson** [Seji64/SniDust#21]
I have a VPS setup in the country I would like to use the service in and SniDust running with `SPOOF_ALL_DOMAINS` set to `true`. I'm using the VPS address as my DNS server both on my device and at my router level (just in case) and have confirmed with `dig` that my VPS is in the answer section. Unfortunately, my home location is still being detected. When I use a paid smart dns service I can access the content without any issues.

Any idea how to start debugging this? Thanks.

**Comment by Seji64** [comment]
I think this is the same "issue" as stated in #7. i really don't know what those paid Services are doing so they are not beeing detected.

Which Site you want to bypass? Maybe they are using webRTC to detect your real IP Adress.

**Comment by stickperson** [comment]
I'm trying to access the Canadian version of dazn.com. The paid service works on my roku TV which I believe does _not_ use webRTC. Not 100% certain though.

**Comment by Seji64** [comment]
WebRTC is just an idea. Probably there 100 other techniqiues...

**Comment by Seji64** [comment]
sniproxy seems currently lacking ipv6 support. (see https://github.com/mosajjal/sniproxy/pull/34). So probably nothing fancy, geo location gets exposed via ipv6

**Comment by stickperson** [comment]
Hm, I'm not seeing any DHCPv6 leases in my router. I can try disabling ipv6 completely to see if that helps.

**Comment by garyfisher** [comment]
> Hm, I'm not seeing any DHCPv6 leases in my router. I can try disabling ipv6 completely to see if that helps.

Did your solution help?

**Comment by stickperson** [comment]
It did not.

**Comment by malex2005** [comment]
> I think this is the same "issue" as stated in #7. i really don't know what those paid Services are doing so they are not beeing detected.
> 
> Which Site you want to bypass? Maybe they are using webRTC to detect your real IP Adress.

Yes, same issue. I guess the streaming provider are blacklisting cloud/vps provider IP ranges. One thing i did not try is test with a paid service. Check the IP and find the local VPS provider used by the paid service. Get a VM from them and try Snidust from there.
